flask_app/mocks.py

```diff
@@ -0,0 +1,47 @@
+SUPPORTED_EXCHANGES = {
+  "Wealthsimple": "ws-crypto-api"
+}
+
+# mock params
+
+MOCK_LOGIN_PARAM = {
+  "exchange": "Wealthsimple",
+  "username": "me@wealthsimple.com",
+  "password": "amazing_api"
+}
+
+MOCK_BALANCES_PARAM = {
+  "user_id": "user-123"
+}
+
+MOCK_TRANSFER_PARAMS = {
+  "account_id": "non-registered-crypto-123",
+  "source_currency": "CAD",
+  "destination_currency": "ETH",
+  "source_amount": 100,
+  "address": "abcde"
+}
+
+# mock responses
+
+MOCK_ACCOUNT_DATA = {
+  "account_id": "non-registered-crypto-123",
+  "currency": "CAD",
+  "balance": 100,
+  "label": "crypto"
+}
+
+MOCK_LOGIN_RESPONSE = {
+  "jwt": "abcde12345",
+  "user_id": "user-123"
+}
+
+MOCK_BALANCES_RESPONSE = {
+  "accounts": [MOCK_ACCOUNT_DATA]
```

Comment:
I think this should be changed to "assets" since that is more consistent with the data model that Tally has and is what we're more interested in. "get me all the assets for this user" A user can have multiple accounts and multiple assets but the atomic (most specific) unit of information we are interested in are the assets.

Reply:
Could you share an example of Tally's model?

Reply:
re: shared in meeting